hooks/generate_scheduler_extender_certs.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from lib.hooks.internal_tls import GenerateCertificateHook, TlsSecret, default_sans
+from lib.module import values as module_values
+from deckhouse import hook
+from typing import Callable
+import common
+
+def main():
+    hook = GenerateCertificateHook(
+        TlsSecret(
+            cn="sds-local-volume-scheduler-extender",
+            name="scheduler-extender-https-certs",
+            sansGenerator=default_sans([
+                "sds-local-volume-scheduler-extender",
+                f"sds-local-volume-scheduler-extender.{common.NAMESPACE}",
+                f"sds-local-volume-scheduler-extender.{common.NAMESPACE}.svc",
+                f"sds-local-volume-scheduler-extender.{common.NAMESPACE}.svc.cluster.local"]),
```

Comment:
Don't need this, also, cluster can use suffix other than "cluster.local"

Reply:
Fixed